boxen/assets/assets.go

```diff
@@ -0,0 +1,7 @@
+package assets
+
+import "embed"
+
+//go:embed scrapli_platforms/*
+// ScrapliPlatformsAssets is the embed FS for the included scrapli platform definition yaml files.
+var ScrapliPlatformsAssets embed.FS
```

Comment:
to support embedded scrapli platform definition files I took the same approach as you did in scrapligo v1

Reply:
now when I look at this, I wonder, should I rather add this platform to scrapligo repo and import it from boxen? That way we have a single point of entry for scrapli definition platforms, both core and community

Reply:
I think either way is fine! it could also just be a repo/gist you host that we can load since the platform def loading thing stole from that work that Michael did w/ textfsm where it will load a url as well!

Reply:
I think I will rework this to be part of the scrapligo, as this will ofc make Checkpoint to be a scrapligo platform, which is a great deal

Reply:
I moved the scrapli definition file to a gist, since I thought it doesn't make a lot of sense to push non core platforms to scrapligo repo